real_time/arduino_src/lib_avr/rbserialmessages/rbsm_headers.txt

```diff
@@ -0,0 +1,11 @@
+RBSM_MID_ENC_TICKS_LAST, 0
```

Comment:
This needs to be compared against high level's header values because high level's implementation of low level's headercheck hasn't been written.